platform/internal/CallChain.h

```diff
@@ -28,7 +28,8 @@ namespace mbed {
 typedef Callback<void()> *pFunctionPointer_t;
 class CallChainLink;
 
-/** \addtogroup platform */
+/** \ingroup mbed-os-internal */
```

Comment:
I don't think it's internal.

Reply:
Do you mean that it should be public? 
At the start of the Epic I asked @kjbracey-arm for a classification of platform APIs betwen public and internal and CallChain was classified as "Private / deprecated (remove?)"
I see this API only used in `InterruptManager.h`